doc/developer/tooling.rst

```diff
@@ -192,18 +192,67 @@ Every release should follow this process:
    archive the new documentation.
 7. Tell everyone!
 
-Module-Level Logging
-====================
-In most of the modules in ``armi``, you will see logging using the ``runLog`` module.
-This is a custom, global logging object provided by the import:
+Logging with runLog
+===================
+ARMI provides a logging tool, ``runLog``, to be used in place of ``print`` for all logging during a
+simulation. It is very easy to use:
 
 .. code-block:: python
 
     from armi import runLog
 
-If you want a logger specific to a single module, say to provide debug logging for only
-one module, that functionality is provided by what might look like a bare Python logging
-import, but is actually calling the same underlying ``armi`` logging tooling:
+    runLog.debug("This will only be seen if you run in debug mode.")
+    runLog.info("Default log level.")
+    runLog.error("The run will die, or the results are invalid.")
+
+.. note::
+    Calling ``runLog.error()`` is not the same as calling Python's ``raise error``; a log statement
+    does not kill a run, or raise an error, it just puts some text in the log.
+
+When an ARMI simulation is run, it will be run at a particular log level. All log messages that are
+at or above that log level will be seen during the simulation and in the final log files. To that
+end, it is important to know how to use each of these log levels:
+
+.. list-table::
+    :widths: 20 20 60
+    :header-rows: 1
+
+    * - Level
+      - Value
+      - When to Use
+    * - debug
+      - 10
+      - This will only be seen if the simulation is run in debug mode.
```

Comment:
Should we reference the `verbosity` and `branchVerobosity` settings here and over these options with an example?

Reply:
Sure!

Reply:
Done!